GoogleForCreators/web-stories-wp
# Text element display: add missing units for padding in Text Display

## Context



## Summary
Ensures that when using `generateParagraphTextStyle`, the unit is always added to the padding. By default, the util does not add any units and it was missing from displaying the text in the editor, causing display issues.


## Relevant Technical Choices



## To-do



## User-facing changes
Padding is displaying correctly now:
<img width="203" alt="Screenshot 2021-12-02 at 15 12 57" src="https://user-images.githubusercontent.com/3294597/144428760-b4e797f6-f542-4704-a3ce-0628faddfd5c.png">


## Testing Instructions






This PR can be tested by following these steps:

1. Add a text element
2. Set Fill as the background mode and set padding (e.g. 15)
3. Verify that padding is added correctly to each side.
4. Test with Highlight mode as well, verify there is padding between the text and the highlighted background

## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #9872
